dashboard/public/robots.txt

```diff
@@ -0,0 +1,4 @@
+User-agent: Googlebot
+Disallow: /tree/
+Disallow: /hardware/
+Disallow: /issue/
```

Comment:
Hmm, why only the Googlebot?

Also, we should allow the top folders `/tree`, `/hardware`, `/issues`, but disallow any sub link to it.